components/bitly/bitly.app.mjs

```diff
@@ -1,11 +1,31 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "bitly",
   propDefinitions: {},
   methods: {
     // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    async createBitlink(data) {
```

Comment:
It would be great if you could create some methods to avoid repeating some code here since some params as the `baseurl` and `headers` could be obtained from a method.
Take a look on `blogger.app.mjs` for better reference

Reply:
Done